partiql-eval/src/main/kotlin/org/partiql/eval/internal/operator/rex/ExprCallDynamic.kt

```diff
@@ -130,7 +130,7 @@ internal class ExprCallDynamic(
 
             init {
                 val lookupsMutable = mutableListOf<CandidateIndex>()
-                val accumulator = mutableListOf<Pair<List<PartiQLValueType>, Candidate>>()
+                val accumulator = mutableListOf<Pair<List<PType>, Candidate>>()
```

Comment:
ACK -- currently the function definitions have PType as the parameter types. Once this is changed (I know you're actively working on this), this can be changed accordingly.